routes/index.js

```diff
@@ -428,6 +428,37 @@ module.exports = function (passport) {
 
   });
 
+
+  router.post('/userManagement/demoteAdmin/:badge', checkAuth, function (req, res) {
+    if (!req.body) {
+      return res.sendStatus(400);
+    }
+    let badgeID = req.params.badge;
+
```

Comment:
Hmmm I was thinking and I think we should now allow the logged in Admin to demote themselves. I think we should add a condition here that goes like this (note there is no 'fade_out'):
```
if (badgeID === req.user.badge) {
  req.flash('error', 'Not allowed to demote yourself from being an Admin');      
  res.redirect('/userManagement/' + badgeID);
  return;
}
```

Reply:
this gonna be before the validateUser function ()?

Reply:
Yes, logically that makes sense

Reply:
Ok I gonna add and test it

Reply:
I guess we need to also add this on the delete route to make sure that the admin can't delete himself

Reply:
Good idea. If you could do that in this branch that would be good.